src/components/ChatList/ChatItemFunc.tsx

```diff
@@ -9,13 +9,16 @@ import StyledButton from '../StyledButton';
 
 // store
 import { useRemoveChatMutation, useUpdateChatMutation } from '../../redux/rtk/chatHistory.api';
+import { createPortal } from 'react-dom';
 
-export default function ChatItemFunc({id, name}: {id: string, name: string}) {
+export default function ChatItemFunc({id, name, isChatPage = false}: {id: string, name: string, isChatPage: boolean}) {
```

Comment:
- [ ] if you use ChatItemFunc for the Chat page you don't need to check "isChatPage" - remove this property